gapic/samplegen/samplegen.py

```diff
@@ -511,3 +512,59 @@ def generate_sample(sample,
                                                 imports=[],
                                                 calling_form=calling_form,
                                                 calling_form_enum=utils.CallingForm)
+
+
+def generate_manifest(fpaths_and_samples,
+                      api_schema,
+                      *,
+                      manifest_time: int = None):
+    """Generate a samplegen manifest for use by sampletest
+
+    Args:
+        fpaths_and_samples (Iterable[Tuple[str, Mapping[str, ?]]]):
+                         The file paths and samples to be listed in the manifest
+
+        api_schema (~api.API): An API schema object.
+        manifest_time (int): Optional. An override for the timestamp in the name of the manifest filename.
+                             Primarily used for testing.
+
+    Returns:
+        Tuple[str, Dict[str,?]]: The filename of the manifest and the manifest data as a dictionary.
+
+    """
+    python_env = {
+        "environment": "python",
+        "bin": "python3",
+        "base_path": "sample/base/directory",
+        "invocation": "{bin} {path} @args",
+    }
+    all_info = {
+        "type": "manifest/samples",
+        "schema_version": 3,
```

Comment:
FYI and reminder to myself: we've been (somewhat inconsistently) using `config_schema_version` because that's what gapic config uses because it's what the API compiler expects, IIUC[*]. However, I am thinking now that for stuff that does not need to go through that, we could be fine using "schema_version" as a shorter alias for the same metadatum.

[*] I cannot find a primary source for this at the moment. I was probably told this and I would assume I had verified it, but we should double-check in the interests of not cargo-culting. @hzyi-google , since you're currently working on gapic-generator, is this something you could check really quickly? Try changing the name of this field in the gapic config and see where what blows up?

Reply:
The 'samplegen manifest files' document uses `schema_version`, not `config_schema_version` in its example. Is it in error?

Reply:
`config_schema_version` is not required by api-compiler. It's just a name used by gapic config. 
 When implementing sample config in the monolith, I used the same name for consistency (though yes, the initial design doc uses `schema_version`). 

I think it's OK to just call it `schema_version` in the manifest file. (After call, the manifest file is generated metadata, not exactly a `config` file).

Reply:
+1 to going with `schema_version` (everywhere). Simpler and shorter, and as @hzyi-google points out, more accurate.